### PR TITLE
[ADD] tests for specific uses cases

### PR DESCRIPTION
**Pull request by hbrunn**
tests for the use cases you already outlined, I'll work on fixing the public holidays after lunch.

This is designed that adding more uses cases should be very convenient, so let's make extensive use of it.

### CODE REVIEW

File: verdigado_attendance/models/hr_attendance.py
Comment (hbrunn) [inline verdigado_attendance/models/hr_attendance.py]: @albig that's the difference between public holidays not working and working. If you agree I'll wrap this override in the standard oca boilerplate and publish it as your module?

Comment (albig) [reply verdigado_attendance/models/hr_attendance.py]: Sound's good.

Comment (hbrunn) [reply verdigado_attendance/models/hr_attendance.py]: https://github.com/OCA/hr-holidays/pull/87

File: repos.yml
Comment (hbrunn) [inline repos.yml:14]: @albig I record the merges I need for things to function here anyways, is this sufficient for you or would you like to have a flat list of PRs to merge somewhere else?

Comment (albig) [reply repos.yml:14]: That's ok for me.

**Review by albig — APPROVED** [review]
Requires `hr_holidays_public_overtime` from https://github.com/OCA/hr-holidays/pull/87

**Comment by hbrunn** [comment]
@albig anything left to do here for merging?

**Comment by albig** [comment]
> @albig anything left to do here for merging?

The PR is fine. But the new requirement for https://github.com/OCA/hr-holidays/pull/87 prevents me from merging (would break my current setup). Any idea how long merging of  https://github.com/OCA/hr-holidays/pull/87  will take?

**Comment by hbrunn** [comment]
just triggered merging this as it got the ready to merge label. But you really need a setup where you don't have to wait for things to be merged, that's a recipe for frustration, and will slow down your development cycle considerably or just stall it forever.

That's why there's git-aggregator, buildout, and all the other ways to stitch together a codebase from a bunch of PRs, I don't think it's possible to do serious development without one of those.

**Comment by hbrunn** [comment]
ah, and as you see I'm not even allowed to do this, so we'll have to wait for some of the hr maintainers to merge it

**Pull request merged**
